Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/IWebView2.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls
+{
+    /// <inheritdoc />
+    public interface IWebView2 : IWebView
```

Comment:
Can we just break the original? I doubt anyone would be implementing their own

Reply:
This would be released with 3.1, which according to semver would not permit such a breaking change. On 4.0 it could be merged to create that single interface.

Reply:
I understand it's a breaking change, but I can't imagine anyone is implementing it. I'm ok with merging them in 4.0. Should be added to the [breaking changes issue](https://github.com/Microsoft/WindowsCommunityToolkit/issues/2178)

Reply:
I've added it to #2178

Reply:
Since our next release is 4.0, does it make sense now to introduce the breaking change?

Reply:
@nmetulev I've merged the interfaces in 5168765